core/src/main/java/hudson/model/Label.java

```diff
@@ -433,6 +436,7 @@ public boolean isEmpty() {
     /*package*/ void reset() {
         nodes = null;
         clouds = null;
+        tiedJobsCount = -1;
```

Comment:
so this will be called every 5 minutes by [trimLabels() periodic task](https://github.com/jenkinsci/jenkins/blob/9c443c8d5bafd63fce574f6d0cf400cd8fe1f124/core/src/main/java/jenkins/model/Jenkins.java#L845-L850)